lib/rgb_matrix/engine.ex

```diff
@@ -243,7 +226,7 @@ defmodule RGBMatrix.Engine do
       %State{state | animation: animation}
       |> inform_configurables()
 
-    {:reply, :ok, state}
+    {:reply, Xebow.update_animation_config(animation), state}
```

Comment:
:warning: (recommended)

`Engine` calling `Xebow` violates the library/application boundary. The flow should be in the other direction.

I recommend Xebow update the config and then pass it to the engine, at which point engine can return `:ok` from this call.

![image](https://user-images.githubusercontent.com/4755047/88870062-3d949200-d1c9-11ea-95d3-6b388ded7134.png)

Reply:
Yeah, that definitely sounds better.